experiment/simulation/js/main.js

```diff
@@ -1 +1,654 @@
-//Your JavaScript goes in here
+'use strict';
+
+document.addEventListener('DOMContentLoaded', function() {
+
+	const menu = document.getElementById('menu');
+	const restartButton = document.getElementById('restart');
+	const instrMsg = document.getElementById('procedure-message');
+
+	menu.addEventListener('change', function() { window.clearTimeout(tmHandle); graphClear(testData[testType].graphs); testType = menu.value; restart(); });
+	restartButton.addEventListener('click', restart);
+
+	function graphClear(graphs) {
+		graphs.forEach((graph, ix) => {
+			document.getElementById('plot' + String(ix)).innerHTML = '';
```

Comment:
String interpolation is preferred over adding 2 strings using the + operator. It is even more useful in this case since you will not have to convert the int to a String explicitly.
Also break this line into 2 with the computation of the id in a separate line;
const plotId = `plot$ix`;
document.getElementById(plotId).innerHTML = '';

This makes it more clear to the reader